core/src/com/unciv/logic/civilization/managers/TechManager.kt

```diff
@@ -459,6 +459,13 @@ class TechManager : IsPartOfGameInfoSerialization {
                             civInfo,
                             triggerNotificationText = "due to entering the [$eraName]"
                         )
+
+            // The unfiltered version
+            for (unique in civInfo.getTriggeredUniques(UniqueType.TriggerUponEnteringEraUnfiltered))
+                UniqueTriggerActivation.triggerUnique(
+                    unique,
+                    civInfo,
+                    triggerNotificationText = "due to entering the new era")
```

Comment:
Reuse the existing text, above lone 460 - more informative, AND translators don't need to add a line